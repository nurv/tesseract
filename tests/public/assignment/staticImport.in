Math.max(Math.cos(Math.PI),Math.sin(Math.E))
static import java.lang.Math;
max(cos(PI),sin(E))
