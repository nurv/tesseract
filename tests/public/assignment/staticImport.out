> 0.41078129050290885
> > 0.41078129050290885
> 